src/main/scala/pipedsl/codegen/bsv/BluespecGeneration.scala

```diff
@@ -397,7 +397,8 @@ object BluespecGeneration {
     private val threadIdVar = BVar(threadIdName, getThreadIdType)
     private val outputData =  BVar("data", translator.toType(mod.ret.getOrElse(TVoid())))
     private val outputQueue = BVar("outputQueue", bsInts.getOutputQType(threadIdVar.typ, outputData.typ))
-
+    //Registers for exceptions
+    private val globalExnFlag = BVar("globalExnFlag", bsInts.getRegType(BBool))
```

Comment:
add underscore "_globalExnFlag" to avoid collision with user supplied variable names